src/frontend/components/component-info/component-info.ts

```diff
@@ -59,12 +59,28 @@ export class ComponentInfo {
     return Object.keys(this.state).length > 0;
   }
 
+  private onTriggerTemplateEvent(listener) {
+    chrome.devtools.inspectedWindow.eval(`
```

Comment:
@riavalon I *think* we should avoid using `inspectedWindow` here, so that it's consistent with how other Augury -> inspected app communication works. Changing properties, and emitting events all happens by the frontend sending a message to the backend, which then executes the operation. That also makes sure our logic is covered by the error reporting exception handler. Does that sound ok to you?